DMX.Core.Api/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=DmxCoreApiDB;Trusted_Connection=True;MultipleActiveResultSets=true"

```

Comment:
Keep this guy in your `appSettings.Development.json`

Reply:
This was needed for the acceptance tests to run in our build pipeline. Is there a better way to store this?

Reply:
No if that's the case that's okay then. good point.